scalafmt-tests/src/test/resources/align/AlignByRightName.stat

```diff
@@ -73,11 +73,11 @@ libraryDependencies ++= Seq(
 libraryDependencies ++= Seq(
 "io.get-coursier" % "interface" %% "0.0.17",
 "com.geirsson1111111" %%% "metaconfig-core" % foo(a %% b) % Test,
-"org.scalacheck" %% "scalacheck" % sclacheckV % Test
+"org.scalacheck" %% "scalacheck" % scalacheckV % Test
```

Comment:
this is a formatting test, and the purpose of it was to demonstrate alignment. therefore, this label needs to be shorter than `foo(a %% b)` above, otherwise there's no alignment difference.

Reply:
`checkV`?